crates/turborepo-filewatch/src/cookie_jar.rs

```diff
@@ -106,7 +106,8 @@ impl CookieJar {
             // dropping the resulting file closes the handle
             trace!("writing cookie {}", cookie_path);
             _ = cookie_path
-                .open_with_options(opts)
+                .ensure_dir()
```

Comment:
The directory deletion could be caused by us or the user. We suggest adding `.turbo` to `.gitignore` so something like `git clean` could cause this. Either way, this PR fixes the behavior if a user deletes the directory while the daemon is running.

Reply:
I believe the reason this might've occurred is because I have a `clean` script defined as `rm -rf .turbo node_modules <...workspace-specific-artifacts>` for every workspace in my monorepo setup. I often run it from monorepo root with `turbo clean && rm -rf .turbo node_modules`. Here `clean` helps me remove unimportant things (artifacts & modules) so I can view branch changes more clearly.